webxr/openxr/mod.rs

```diff
@@ -436,10 +440,23 @@ impl Device for OpenXrDevice {
             .locate(&self.space, self.frame_state.predicted_display_time)
             .unwrap();
 
+        let pose_valid = location.location_flags.intersects(
+            SpaceLocationFlags::ORIENTATION_VALID | SpaceLocationFlags::ORIENTATION_VALID,
```

Comment:
This looks weird - are there supposed to be two different flags here?

Reply:
Yes

Reply:
Fixed.